templates/vrrp_instance-footer.erb

```diff
@@ -0,0 +1,81 @@
+  <%- if @track_script or @collect_exported -%>
```

Comment:
could you add a comment at the top like `# THIS FILE IS MANAGED BY PUPPET`?

Are you interested in converting this to an epp template? That's our prefered template language (but erb works as well).